CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+# v0.0.44
+
+## Announcing Akri v0.0.44!
+Akri v0.0.44 is a pre-release of Akri.
+
+To find out more about Akri, check out our [README](https://github.com/deislabs/akri/blob/main/README.md) and start [contributing](https://github.com/deislabs/akri/blob/main/docs/contributing.md) today!
+
+## New Features
```

Comment:
Was the basic RBAC support part of 0.0.35?

Reply:
yeah, just barely.  you can view 0.0.35 release commits here: https://github.com/deislabs/akri/commits/v0.0.35

Reply:
Nice, thanks.